Resources/Prototypes/Recipes/Lathes/cooking.yml

```diff
@@ -35,6 +35,20 @@
   materials:
     Glass: 50
 
+- type: latheRecipe
+  id: DrinkShotGlass
+  result: DrinkShotGlass
+  completetime: 0.4
+  materials:
+    Glass: 25
```

Comment:
This needs to be more expensive than `DrinkShotGlass`'s material composition of 25 glass (or alternatively, the material composition of `DrinkShotGlass` can be reduced. Just be sure to mind the material composition of the glass shards!)

Reply:
glass shards don't seem to _have_ a material composition, but I'll just move all glasses to 1 sheet to align with the beaker